schema/dpns-contract-documents.json

```diff
@@ -0,0 +1,127 @@
+{
+  "domain": {
+    "indices": [
+      {
+        "properties": [
+          {
+            "normalizedParentDomainName": "asc"
+          },
+          {
+            "normalizedLabel": "asc"
+          }
+        ],
+        "unique": true
+      },
+      {
+        "properties": [
+          {
+            "records.dashUniqueIdentityId": "asc"
+          }
+        ],
+        "unique": true
+      }
+    ],
+    "properties": {
+      "label": {
+        "type": "string",
+        "pattern": "^((?!-)[a-zA-Z0-9-]{0,62}[a-zA-Z0-9])$",
+        "minLength": 3,
+        "maxLength": 63,
+        "description": "Domain label. e.g. 'Bob'."
+      },
+      "normalizedLabel": {
+        "type": "string",
+        "pattern": "^((?!-)[a-z0-9-]{0,62}[a-z0-9])$",
+        "maxLength": 63,
+        "description": "Domain label in a lower case for case insensitive uniqueness validation. e.g. 'bob'",
+        "$comment": "This property will be deprecated due to case insensitive indices"
```

Comment:
What's makes us not deprecating it right now ? @shumkov

Reply:
We don't have such indices atm :)